package/yast2-vm.changes

```diff
@@ -3,6 +3,8 @@ Wed Oct 11 14:59:40 UTC 2017 - aginies@suse.com
 
 - use new pattern name for SLES (same as TW)
 - Remove SLED as this is no more needed
+- BSC1055459 
```

Comment:
AFAIK OBS strictly checks for bsc#1234. I know that it requires the '#' and no blanks anywhere, otherwise the changelog check bot will reject the submit request. Not sure if it also accepts a capital "BSC". So this should be

    bsc#1055459